build/victoriametrics/files/victoriametrics-template.xml

```diff
@@ -0,0 +1,162 @@
+                        restart_on="none"
+                        type="service">
+                <service_fmri value="svc:/system/filesystem/local:default" />
+            </dependency>
+            <method_context>
+                <method_credential user="$(USER)"
+                                   group="$(GROUP)"
+                                   privileges="basic,net_privaddr" />
+                <method_environment>
+                    <envvar name="VM_storageDataPath"
+                            value="/$(DATA)" />
+                </method_environment>
+            </method_context>
+
+            <exec_method type="method"
+                         name="start"
+                         exec="%{config/exec} -envflag.enable -envflag.prefix=VM_"
+                         timeout_seconds="60"></exec_method>
+
+            <exec_method type="method"
+                         name="stop"
+                         exec=":kill"
+                         timeout_seconds="300" />
+
+            <property_group name="config"
+                            type="application">
+                <propval name="exec"
+                         type="astring"
+                         value="/$(PREFIX)/bin/$(VM)" />
+            </property_group>
+
+            <property_group name="startd"
+                            type="framework">
+                <propval name="duration"
+                         type="astring"
+                         value="child" />
+                <propval name="ignore_error"
+                         type="astring"
+                         value="core,signal" />
+            </property_group>
+
+            <template>
+                <common_name>
+                    <loctext xml:lang="C">VictoriaMetrics server
+                    $(VERSION)</loctext>
+                </common_name>
+            </template>
+
+        </instance>
+
+        <instance name="vmagent"
+                  enabled="false">
+
+            <dependency name="loopback"
+                        grouping="require_any"
+                        restart_on="error"
+                        type="service">
+                <service_fmri value="svc:/network/loopback" />
+            </dependency>
+
+            <dependency name="network"
+                        grouping="optional_all"
+                        restart_on="error"
+                        type="service">
+                <service_fmri value="svc:/milestone/network" />
+            </dependency>
+
+            <dependency name="filesystem_local"
+                        grouping="require_all"
+                        restart_on="none"
+                        type="service">
+                <service_fmri value="svc:/system/filesystem/local:default" />
+            </dependency>
+            <method_context>
+                <method_credential user="$(USER)"
+                                   group="$(GROUP)"
+                                   privileges="basic,net_privaddr" />
```

Comment:
Does this service need to bind to TCP/UDP ports below 1024? If not, please drop the `net_privaddr` privilege.
Same for the other services.

Reply:
Good question, they do not. By default both services bind to 8428 and 8429.
Privilege dropped.